modules/grpc/1.62.1.bcr.2/patches/py_grpc_library.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/bazel/python_rules.bzl b/bazel/python_rules.bzl
```

Comment:
Can we upstream this patch?

Reply:
I sent a PR: https://github.com/grpc/grpc/pull/36843